hpOneView/common.py

```diff
@@ -1827,3 +1827,19 @@ def transform_list_to_dict(list):
             ret[str(value)] = True
 
     return ret
+
+
+def extract_id_from_uri(id_or_uri):
+    """
+    Extract ID from the end of the URI
+
+    Args:
+        id_or_uri: ID or URI of the OneView resources.
+
+    Returns:
+        str: The string founded after the last "/"
```

Comment:
I've created this in other branch. To avoid conflict, I suggest to not change it now. Anyway, I still prefer the "/" character.